Python/Product/Analysis/Intellisense/ImportedModuleNameWalker.cs

```diff
@@ -18,41 +18,83 @@
 using System.Linq;
 using Microsoft.PythonTools.Analysis;
 using Microsoft.PythonTools.Analysis.Infrastructure;
+using Microsoft.PythonTools.Analysis.LanguageServer;
 using Microsoft.PythonTools.Parsing.Ast;
 
 namespace Microsoft.PythonTools.Intellisense {
+    sealed class NamedLocation {
+        public string Name { get; set; }
+        public Range Range { get; set; }
```

Comment:
Use `SourceSpan` while we're fully within the analyzer. `Range` is really only for LSP

Reply:
Sure

Reply:
Shall we unify this at some point? Something like `ISourceLocation` . Comes handy when searching trees (provided nodes implement some `ITextRange`.